app/concepts/api/v1/sessions/operation/create.rb

```diff
@@ -0,0 +1,23 @@
+module Api::V1
+  module Sessions::Operation
+    class Create < Trailblazer::Operation
+      step :model!
+      step :authenticate!
+      step :login
+
+      def model!(ctx, params:, **)
+        ctx['model'] = User.find_by!(username: params[:username])
+      end
+
+      def authenticate!(ctx, params:, **)
```

Comment:
do not raise errors in operations
use `failure` step to handle the case when the password is incorrect